### ISSUE TITLE
Playing audio files which are stored in an expansion file

### ISSUE THREAD

**Issue by JohnWeidner** [moust/phonegap-xapkreader#21]
Will this plugin work for playing an audio file that is stored in an expansion file?    If so, do you have any sample code that you could share?   Thanks!!!

**Comment by rudiminty** [comment]
No, it will not. I’ve created some custom code on this project in the past that will allow for extracting an audio file (any file) from the APK and then return the path of that file to be played by the media player.
